ec/src/glv.rs

```diff
@@ -0,0 +1,35 @@
+use crate::ModelParameters;
+
+/// The GLV parameters that are useful to compute the endomorphism
+/// and scalar decomposition.
+pub trait GLVParameters: Send + Sync + 'static + ModelParameters {
+    type CurveAffine;
+    type CurveProjective;
+
+    // phi(P) = lambda*P for all P
+    // constants that are used to calculate phi(P)
+    const COEFF_A1: Self::BaseField;
```

Comment:
Can there be a docstring explaining what is A1, A2, A3, B1, B2 etc?

Reply:
Will do!

Reply:
The docstrings are added